www/source/stylesheets/_docs.scss

```diff
@@ -118,25 +118,35 @@ $docs-xlarge-breakpoint: rem-calc(1060);
   }
 }
 
-// Quicklinks
-.docs--quicklink {
-  padding: 0;
+//Getting Started
+.get-started--boxes {
 
-  .docs--quicklink--cards,
-  .docs--quicklink--card--wrapper {
+  .card {
+    @include card;
     display: flex;
-  }
+    padding: rem-calc(24)rem-calc(24)rem-calc(0)rem-calc(24);
+    position: relative;
+    border-radius: $global-radius;
+
+    p {
+      color: $hab-gray-dark;
+    }
+
+    > p:last-child {
+      margin-bottom: 0;
+    }
+
+    }
```

Comment:
Super minor, but this is tabbed in a couple of spaces too many. (It's easier to follow the nesting visually when these match up properly.)